content/en/client/authorized.md

```diff
@@ -77,48 +91,47 @@ With our OAuth token for the authorized user, we can now perform any action as t
-* See [/api/v1/follow_requests]({{< relref "methods/follow_requests" >}}) for handling follow requests.
-* See [/api/v1/mutes]({{< relref "methods/mutes" >}}) for listing mutes.
-* See [/api/v1/blocks]({{< relref "methods/blocks" >}}) for listing blocks.
+- See [/api/v1/accounts]({{< relref "methods/accounts" >}}) for performing actions on other users.
+- See [/api/v1/follow_requests]({{< relref "methods/follow_requests" >}}) for handling follow requests.
+- See [/api/v1/mutes]({{< relref "methods/mutes" >}}) for listing mutes.
+- See [/api/v1/blocks]({{< relref "methods/blocks" >}}) for listing blocks.
 
 ### Receive notifications {#notifications}
 
-* See [/api/v1/notifications]({{< relref "methods/notifications" >}}) for managing a user's notifications.
-* See [/api/v1/push]({{< relref "methods/push" >}}) for subscribing to push notifications.
+- See [/api/v1/notifications]({{< relref "methods/notifications" >}}) for managing a user's notifications.
+- See [/api/v1/push]({{< relref "methods/push" >}}) for subscribing to push notifications.
 
 ### Discovery features {#discovery}
 
-* See [/api/v2/search]({{< relref "methods/search#v2" >}}) for querying resources.
-* See [/api/v1/suggestions]({{< relref "methods/suggestions" >}}) for suggested accounts to follow.
+- See [/api/v2/search]({{< relref "methods/search#v2" >}}) for querying resources.
+- See [/api/v1/suggestions]({{< relref "methods/suggestions" >}}) for suggested accounts to follow.
 
 ### User safety features {#safety}
 
-* See [/api/v1/filters]({{< relref "methods/filters" >}}) for managing filtered keywords.
-* See [/api/v1/domain_blocks]({{< relref "methods/domain_blocks" >}}) for managing blocked domains.
-* See [/api/v1/reports]({{< relref "methods/reports" >}}) for creating reports.
-* See [/api/v1/admin]({{< relref "methods/admin" >}}) for moderator actions.
+- See [/api/v1/filters]({{< relref "methods/filters" >}}) for managing filtered keywords.
+- See [/api/v1/domain_blocks]({{< relref "methods/domain_blocks" >}}) for managing blocked domains.
+- See [/api/v1/reports]({{< relref "methods/reports" >}}) for creating reports.
+- See [/api/v1/admin]({{< relref "methods/admin" >}}) for moderator actions.
 
 ### Manage account info {#manage}
 
-* See [/api/v1/endorsements]({{< relref "methods/endorsements" >}}) for managing a user profile's featured accounts.
-* See [/api/v1/featured_tags]({{< relref "methods/featured_tags" >}}) for managing a user profile's featured hashtags.
-* See [/api/v1/preferences]({{< relref "methods/preferences" >}}) for reading user preferences.
-
+- See [/api/v1/endorsements]({{< relref "methods/endorsements" >}}) for managing a user profile's featured accounts.
+- See [/api/v1/featured_tags]({{< relref "methods/featured_tags" >}}) for managing a user profile's featured hashtags.
+- See [/api/v1/preferences]({{< relref "methods/preferences" >}}) for reading user preferences.
```

Comment:
It would potentially be useful here to run prettier in separate branch/PR, get that merged first, rebase this, etc.

Reply:
I think it's "fine" for now? But do a follow up to run prettier; I'd rather not be rebasing this branch since there's now quite a lot of commits and rebasing would take quite a long time.